src/js/intlTelInput.js

```diff
@@ -938,6 +941,9 @@ Plugin.prototype = {
       }
       this.dropdown.detach();
     }
+
+    this._triggerCloseCountryDropdown();
```

Comment:
No need to create a new function for this just yet, if it's only called in one place, and it's just a one-liner anyway.